segmentation_labeling_app/rois/rois.py

```diff
@@ -0,0 +1,215 @@
+"""
+This module is used for loading the ROIs from LIMs DB tables and
+contains pre processing methods for ROIs loaded from LIMs. These methods
+are used to define drawing parameters as well as other ROI class
+methods useful for post processing required to display to end user.
+"""
+from typing import List, Tuple, Union
+
+from scipy.sparse import coo_matrix
+import scipy.ndimage
+import numpy as np
+import cv2
+
+import segmentation_labeling_app.utils.query_utils as query_utils
+from segmentation_labeling_app.utils.math_utils import get_magnitude
+
+stroke_weight = 1.125
+
+
+class ROI:
+
+    def __init__(self,
+                 coo_rows: Union[np.array, List[int]],
+                 coo_cols: Union[np.array, List[int]],
+                 coo_data: Union[np.array, List[float]],
+                 image_shape: Tuple[int, int],
+                 segmentation_id: int,
+                 roi_id: int):
+        self.coo_rows = coo_rows
+        self.coo_cols = coo_cols
+        self.coo_data = coo_data
```

Comment:
your `coo_*` properties are going to be duplicated between `ROI` and `ROI._sparse_coo`. Maybe not in space, if they are by reference, but, it might be more clear if the matrix is only described in one place.